pom.xml

```diff
@@ -113,6 +113,16 @@
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-csv</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.poi</groupId>
+            <artifactId>poi</artifactId>
+            <version>5.1.0</version>
```

Comment:
Could you add a variable for the poi version? It'd be declared up around line 18, and then you'd use it for the two new dependencies to make sure they stay synced with each other